app/forms/schools/add_participants/wizard_steps/email_step.rb

```diff
@@ -31,8 +31,6 @@ def next_step
             end
           elsif wizard.sit_adding_themself_as_mentor?
             :yourself
-          elsif wizard.needs_to_confirm_start_term?
-            :start_term
           elsif wizard.needs_to_choose_a_mentor?
```

Comment:
```suggestion
          elsif wizard.needs_to_confirm_start_term? && !Cohort.within_next_registration_period?
             :cannot_add_registration_not_yet_open
          elsif wizard.needs_to_confirm_start_term? && wizard.need_training_setup?
             :need_training_setup
          elsif wizard.needs_to_choose_a_mentor?
```

Reply:
This is to keep the logic of steps that would go after the one we have removed.

Reply:
Ah I see, there's a case where we aren't in the registration period yet so we need to respond with the `:cannot_add_registration_not_yet_open` step, but this doesn't have anything to do with the `needs_to_confirm_start_term?` method does it? Given two of the conditional clauses in the `needs_to_confirm_...` method are redundant now ie. `(mentor_participant? || induction_start_date.blank?)` we can combine like so here:

```ruby
if !Cohort.within_next_registration_period? && !Cohort.within_automatic_assignment_period?
  :cannot_add_registration_not_yet_open
```

Reply:
Have renamed this method and call as suggested to preserve routing to the terminal _registration not yet open_ step